index.bs

```diff
@@ -166,6 +166,41 @@ interface DetectedBarcode {
 Example implementations of Barcode/QR code detection are e.g. <a href="https://developers.google.com/android/reference/com/google/android/gms/vision/barcode/package-summary">Google Play Services</a> or <a href="https://developer.apple.com/reference/coreimage/ciqrcodefeature?language=objc">Apple's CICRCodeFeature</a>.
 </div>
 
+## Text Detection API
+
+{{TextDetector}} represents an underlying accelerated platform's component for detection in images of text.  It provides a single {{TextDetector/detect()}} operation on an {{ImageBitmapSource}} of which the result is a Promise.  This method MUST reject this Promise in the cases detailed in [[#image-sources-for-detection]]; otherwise it MAY queue a task using the OS/Platform resources to resolve the Promise with a sequence of {{DetectedText}}s, each one essentially consisting on and delimited by a {{DetectedText/boundingBox}}.
+
+<pre class="idl">
+[
+    Constructor,
+    Exposed=(Window,Worker),
+] interface TextDetector {
+    Promise&lt;sequence&lt;DetectedText&gt;&gt; detect(ImageBitmapSource image);
+};
+</pre>
+
+<dl class="domintro">
+  <dt><dfn method for="TextDetector"><code>detect(ImageBitmapSource image)</code></dfn></dt>
+  <dd>Tries to detect text blocks in the {{ImageBitmapSource}} |image|.</dd>
+</dl>
+
+<pre class="idl">
+[
+    Constructor,
+] interface DetectedText {
+    [SameObject] readonly attribute DOMRect boundingBox;
```

Comment:
I'd say we need here a `readonly attribute DOMString rawValue;` or 
similar otherwise how's the API going to communicate the detected
feature?  Make sure to refer to it in this section's intro paragraph.